common/static/coffee/src/discussion/views/discussion_thread_edit_view.js

```diff
@@ -75,7 +81,9 @@
                         this.$('.edit-post-title').val('').attr('prev-text', '');
                         this.$('.edit-post-body textarea').val('').attr('prev-text', '');
                         this.$('.wmd-preview p').html('');
-                        postData.courseware_title = this.topicView.getFullTopicName();
+                        if (this.topicView) {
```

Comment:
It looks like postData is passed to the AJAX call, but then is subsequently updated and then set into the model. This does seem very confusing...